crd-generator/src/main/java/io/strimzi/crdgenerator/DocGenerator.java

```diff
@@ -538,7 +538,7 @@ public static void main(String[] args) throws IOException, ClassNotFoundExceptio
                 .append(".").append(NL);
             writer.append("// To change this documentation you need to edit the Java sources.").append(NL);
             writer.append(NL);
-            DocGenerator dg = new DocGenerator(crApiVersion, 3, classes, writer, linker);
+            DocGenerator dg = new DocGenerator(crApiVersion, 1, classes, writer, linker);
```

Comment:
Looks like these changed broke the tests. So you would need to update the _test_ asciidoc as well.